src/references/chain-assets.json

```diff
@@ -16,368 +16,5 @@
       },
       "quantity": 0
     }
-  ],
-  "optimism": [
```

Comment:
Looks like basing `optimismExplorerInit` around `chain-assets.json` no longer makes sense. That should be addressed at some point, but, imo, you should leave this as `"optimism": []` to get the fix out and get passed the linter.

Reply:
good call!